composer.json

```diff
@@ -28,29 +28,23 @@
     },
     "sort-packages": true
   },
-  "extra": {
-    "downloads": {
-      "phpstan": {
-        "url": "https://github.com/phpstan/phpstan/releases/latest/download/phpstan.phar",
-        "path": "vendor/bin/phpstan",
-        "type": "phar"
-      }
-    }
-  },
   "autoload": {
     "psr-4": {
       "Google\\Web_Stories\\": "includes"
     }
   },
   "autoload-dev": {
     "psr-4": {
-      "Google\\Web_Stories\\Tests\\": "tests/phpunit/includes",
-      "PHPStan\\WordPress\\": "tests/phpstan/src"
+      "Google\\Web_Stories\\Tests\\": "tests/phpunit/includes"
     }
   },
   "prefer-stable": true,
   "scripts": {
     "phpcs": "phpcs",
-    "phpstan": "phpstan analyse --memory-limit=512M"
+    "phpstan": [
+      "mkdir php7 || echo",
+      "composer --working-dir=php7 show --path szepeviktor/phpstan-wordpress || composer --working-dir=php7 require --dev --ignore-platform-reqs szepeviktor/phpstan-wordpress",
```

Comment:
I'm still on the fence whether I like this or not.

It's certainly clever and does the job, but it makes it less obvious for maintainers which dependencies are installed and in which version.

I feel like I'll have to think about this a bit more.

Reply:
we can add `:^0.7.0` ...

Reply:
@swissspidy Please close your eyes 🙄 for a moment and merge this we can always talk in the next issue/PR.